src/components/Home/Index.jsx

```diff
@@ -67,21 +67,28 @@ function Index() {
           </p>
           <p className="p-description">
             The Terasology project was born from a Minecraft-inspired tech demo
-            and is becoming a stable platform for various types of gameplay
-            settings in a voxel world. The creators and maintainers are a
-            diverse mix of software developers, designers, game testers, graphic
-            artists, and musicians. We encourage others to join!
+            in 2011. Over the years it has been a playground for different kinds
+            of technical experiments and proof of concepts. The community
+            strives to make Terasology a stable platform for various types of
+            gameplay settings in a voxel world. The contributors are a diverse
```

Comment:
Is it on purpose that we have the sentence on "diverse mix" both here and further down on the page now?